test/integration/specs/kbaseFeatureSet_test.js

```diff
@@ -0,0 +1,105 @@
+/*global describe, it, browser, expect, $, afterEach*/
+const {login, openNarrative} = require('../wdioUtils.js');
+
+const TEST_CASE1 = {
+    narrativeId: 56638,
```

Comment:
Is this a public narrative?

Reply:
no; it is owned by kbaseui test, shared with narrative test.